packages/htmljs/html.js

```diff
@@ -1,28 +1,71 @@
+export const Tag = function () {
+};
 
-export const Tag = function () {};
 Tag.prototype.tagName = ''; // this will be set per Tag subclass
 Tag.prototype.attrs = null;
 Tag.prototype.children = Object.freeze ? Object.freeze([]) : [];
 Tag.prototype.htmljsType = Tag.htmljsType = ['Tag'];
 
+export function isConstructedObject(x) {
```

Comment:
linter gives an error if the function is called before definition. I didn't like this but the rule is a rule.

Reply:
Words are spoken... to be broken... :D -> I think it's ok to skip this linter rule in case of higher goods, eg. Reviewers comfort :D / being able to review the changes effectively. IMHO?